### PR TITLE
#82 add shell script for compiling and running tests with substrate

### PR DESCRIPTION
**Pull request by tiainen**
Adds a shell script that can compile and run test projects with a locally compiled version of substrate.

### CODE REVIEW

File: run-tests.sh [hunk 1]
```diff
@@ -0,0 +1,100 @@
+#!/bin/bash -ex
```
[the remark this answers is not in the document]
Comment (tiainen) [reply run-tests.sh]: It's not required, so I removed it.

File: run-tests.sh [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  if [ ! -f libs/javafx/javafx-${javafxModule}-${JAVAFX_VERSION}-${JAVAFX_PLATFORM}.jar ]; then
+    wget -O libs/javafx/javafx-${javafxModule}-${JAVAFX_VERSION}-${JAVAFX_PLATFORM}.jar https://repo1.maven.org/maven2/org/openjfx/javafx-${javafxModule}/${JAVAFX_VERSION}/javafx-${javafxModule}-${JAVAFX_VERSION}-${JAVAFX_PLATFORM}.jar
+  fi
+done
+echo "Downloading JavaFX jars completed!"
+
+echo "Downloading JavaFX static SDK ${JAVAFX_STATIC_VERSION}-${JAVAFX_STATIC_PLATFORM}..."
+mkdir -p libs/javafxstatic
+if [ ! -f libs/javafxstatic/openjfx-${JAVAFX_STATIC_VERSION}-${JAVAFX_STATIC_PLATFORM}-static.zip ]; then
+  wget -O libs/javafxstatic/openjfx-${JAVAFX_STATIC_VERSION}-${JAVAFX_STATIC_PLATFORM}-static.zip https://download2.gluonhq.com/substrate/javafxstaticsdk/openjfx-${JAVAFX_STATIC_VERSION}-${JAVAFX_STATIC_PLATFORM}-static.zip
```
[the remark this answers is not in the document]
Comment (tiainen) [reply run-tests.sh]: Yes, we were considering to do that. That's something that we will put on our todo list.

File: run-tests.sh [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  unzip -d libs/javafxstatic libs/javafxstatic/openjfx-${JAVAFX_STATIC_VERSION}-${JAVAFX_STATIC_PLATFORM}-static.zip
+fi
+echo "Downloading JavaFX static SDK completed!"
+
+echo "Downloading substrate dependencies..."
+mkdir -p libs/substrate
+if [ ! -f libs/substrate/dd-plist-1.22.jar ]; then
+  wget -O libs/substrate/dd-plist-1.22.jar https://repo1.maven.org/maven2/com/googlecode/plist/dd-plist/1.22/dd-plist-1.22.jar
```
[the remark this answers is not in the document]
Comment (tiainen) [reply run-tests.sh]: I've changed this to use `mvn dependency:get` instead. You can probably override the default maven repository inside settings.xml?

[the remark this answers is not in the document]
Comment (vjovanov) [reply run-tests.sh]: Yes, that is what I intend to do.

**Pull request merged**